### PR TITLE
design considerations

### LINKED ISSUE #54: add small section on Design Considerations

### PR DESCRIPTION
**Pull request by paulbastian**
Closes #54 

## 📑 Description
This PR adds a section on design considerations.

## Preview Link

[click here for rendered preview of PR](https://github.com/<repo-name>/<branch-name>/<draft-name>.html)

### CODE REVIEW

File: draft-looker-oauth-jwt-cwt-status-list.md [hunk 1]
```diff
@@ -65,7 +65,18 @@ The following diagram depicts the basic conceptual relationship.
...
+## Design Considerations
+
+The decisions taken in this specification aim to achieve the following design goals:
+
+* the specification shall favor a simple and easy to understand concept
+* the specification shall be easy, fast and secure to implement in all major programming languages
+* the specification shall be optimized to support the most common use cases and avoid unneccessary complexity of corner cases
+* the Status List shall scale up to millions of tokens to support large scale government or enterprise use cases
+* the Status List shall enable caching policies and offline support
+* the specification shall support JSON and CBOR based tokens
```
Comment (c2bo) [inline draft-looker-oauth-jwt-cwt-status-list.md:79]:
```suggestion
* the specification shall support JSON and CBOR based tokens
* the specification shall not specify token specific mechanisms like validation or key resolution
```

**Review by c2bo — APPROVED** [review]

File: draft-looker-oauth-jwt-cwt-status-list.md (hunk 1 above)
Comment (paulbastian) [reply draft-looker-oauth-jwt-cwt-status-list.md:79]: I agree with key resolution, but token specific mechanisms may be an overstatement, as we define JWT and CWT specific representations

Comment (c2bo) [reply draft-looker-oauth-jwt-cwt-status-list.md:79]: updated it - does this version look better for you?

File: draft-ietf-oauth-status-list.md [hunk 2]
```diff
@@ -66,7 +66,18 @@ The following diagram depicts the basic conceptual relationship.
...
+## Design Considerations
+
+The decisions taken in this specification aim to achieve the following design goals:
+
+* the specification shall favor a simple and easy to understand concept
+* the specification shall be easy, fast and secure to implement in all major programming languages
+* the specification shall be optimized to support the most common use cases and avoid unneccessary complexity of corner cases
+* the Status List shall scale up to millions of tokens to support large scale government or enterprise use cases
+* the Status List shall enable caching policies and offline support
+* the specification shall support JSON and CBOR based tokens
```
Comment (paulbastian) [inline draft-ietf-oauth-status-list.md:80]:
```suggestion
* the specification shall support JSON and CBOR based tokens
* the specification shall not specify key resolution or trust frameworks
```

**Comment by paulbastian** [comment]
I had to force push changes as the branch originated before the renaming of the file and my git merge messed things up

**Comment by paulbastian** [comment]
@tplooker could you please review this

**Review by tplooker — APPROVED** [review]

**Pull request merged**